backend/node/base.py

```diff
@@ -89,27 +89,14 @@ async def run_contract(
     def get_contract_data(
         self, code: str, state: str, method_name: str, method_args: list
     ):
-        output_buffer = io.StringIO()
-
-        result = GenVM.get_contract_data(
+        result = self.genvm.get_contract_data(
             code,
             state,
             method_name,
             method_args,
             self.contract_snapshot_factory,
-            output_buffer,
         )
 
-        if self.genvm.contract_runner.mode == ExecutionMode.LEADER:
```

Comment:
@AgustinRamiroDiaz @denishacquin could you please point to the place where this is being done now?

Reply:
It was the fastest way at the moment, but I think that we'll eventually move all logging outside of the GenVM